docs/dns/domains.rst

```diff
@@ -115,6 +116,25 @@ Field details:
     write operations that did not trigger publication, such as rewriting an
     RRset with identical values.
 
+``zonefile``
+    :Access mode: write-only
+
+    Optionally, includes a string in zonefile format with zone data to be
+    imported during domain creation.
+
+    Note that not everything given in the zonefile will be imported. Record
+    types that are `automatically managed by the deSEC API`_ such as RRSIG,
+    CDNSKEY, CDS, etc. will be silently ignored. Records with owner names
+    outside of the domain that is created will also be silently ignored.
```

Comment:
`owner name` is too technical IMO. `Records with names that fall outside the domain that is created`

Reply:
done